src/metabase/analytics/snowplow.clj

```diff
@@ -76,7 +75,7 @@
                   (let [value (or (first-user-creation) (t/offset-date-time))]
                     (setting/set-value-of-type! :timestamp :instance-creation value)
                     (track-event! ::new-instance-created)))
-                (setting/get-value-of-type :timestamp :instance-creation)))
+                (u.date/format-rfc3339 (setting/get-value-of-type :timestamp :instance-creation))))
```

Comment:
Format it here because FE uses the value of this setting to send snowplow events from UI.